src/Symfony/Component/Yaml/Inline.php

```diff
@@ -560,13 +531,15 @@ private static function evaluateScalar(string $scalar, int $flags, array $refere
                 throw new ParseException('A reference must contain at least one character.', self::$parsedLineNumber + 1, $value, self::$parsedFilename);
             }
 
-            if (!\array_key_exists($value, $references)) {
+            if (!isset($references[$value])) {
```

Comment:
I think the reference values can be `null`, so we need `array_key_exists` here:
https://yaml-online-parser.appspot.com/?yaml=-+%26foo+null%0A-+*foo%0A&type=json

btw, since php 7.3 `array_key_exists` is really fast: https://github.com/php/php-src/pull/3360

Reply:
Reverting to `array_key_exists` and thanks for pointing that one out, I wasn't aware this had been optimised in opcode.